twelvedata/src/endpoint/price.ts

```diff
@@ -0,0 +1,31 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, Config } from '@chainlink/types'
+import { NAME as AdapterName } from '../config'
+
+export const NAME = 'price'
+
+const customError = (data: any) => data.Response === 'Error'
+
+const customParams = {
+  base: ['base', 'from', 'coin', 'market'],
+}
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const symbol = validator.overrideSymbol(AdapterName).toUpperCase()
+
+  const url = `price?symbol=${symbol}&apikey=${config.apiKey}`
```

Comment:
You could use `params` on options to pass these in. Might look a little cleaner.

So in this case it would be:
```
const url = 'price'

const params = {
symbol,
apikey: config.apiKey
}

const options = {
...config.api,
params,
url,
}
```